src/logic/azext_logic/generated/_params.py

```diff
@@ -0,0 +1,133 @@
+            self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type, help='The resource tags.')
+        c.argument('sku', action=AddSku, nargs='+', help='The sku.')
+        c.argument('integration_service_environment', arg_type=CLIArgumentType(options_list=['--integration-service-env'
+                                                                                             'ironment'], help='The integration se'
+                                                                               'rvice environment. See README.md For more information'))
+        c.argument('state', arg_type=get_enum_type(['NotSpecified', 'Completed', 'Enabled', 'Disabled', 'Deleted', 'Sus'
+                                                    'pended']), help='The workflow state.')
+
+    with self.argument_context('logic integration-account update') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The integration account name.')
+        c.argument('tags', tags_type, help='The resource tags.')
+        c.argument('sku', action=AddSku, nargs='+', help='The sku.')
+        c.argument('integration_service_environment', arg_type=CLIArgumentType(options_list=['--integration-service-env'
+                                                                                             'ironment'], help='The integration se'
+                                                                               'rvice environment. See README.md For more information'))
+        c.argument('state', arg_type=get_enum_type(['NotSpecified', 'Completed', 'Enabled', 'Disabled', 'Deleted', 'Sus'
+                                                    'pended']), help='The workflow state.')
+
+    with self.argument_context('logic integration-account delete') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The integration account name.')
+
+    with self.argument_context('logic integration-account import') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The integration account name.')
+        c.argument('input_path', type=validate_file_or_dict,
+                   help='Path to a intergration-account JSON file', completer=FilesCompleter())
+        c.argument('location', arg_type=get_location_type(
+            self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type, help='The resource tags.')
+        c.argument('sku', type=str, help='The integration account sku.')
```

Comment:
In the custom method, the values for these 3 options are not used. You can remove them.

Reply:
The value is used if it is not defined in the imported JSON:

 location=input_path.get('location', location),
tags=input_path.get('tags', tags),
 sku=input_path.get('sku', sku),